.github/workflows/cherry-pick-milestoned-prs.yml

```diff
@@ -0,0 +1,406 @@
+              `Cherry picking was successful for \`${targetBranch}\`. Please merge the following PR: [${cherryPickPrTitle}](${cherryPickPrUrl})`;
+
+            await github.rest.issues.createComment({
+              owner,
+              repo,
+              issue_number: parseInt('${{ needs.prepare.outputs.pr_number }}'),
+              body
+            });
+      
+      # Notify Slack about successful cherry-pick to next branch
+      - name: Notify Slack on success
+        uses: archive/github-actions-slack@c643e5093620d65506466f2c9b317d5d29a5e517 # v2.10.1
+        env:
+          CHERRY_PICK_PR_URL: https://github.com/${{ github.repository }}/pull/${{ needs.cherry-pick-next.outputs.cherry_pick_pr_number }}
+          SOURCE_PR_URL: ${{ github.event.pull_request.html_url }}
+          SOURCE_PR_NUMBER: ${{ github.event.pull_request.number }}
+          SOURCE_PR_TITLE: ${{ github.event.pull_request.title }}
+        with:
+          slack-bot-user-oauth-access-token: ${{ secrets.CODE_FREEZE_BOT_TOKEN }}
+          slack-channel: ${{ secrets.WOO_RELEASE_SLACK_NOTIFICATION_CHANNEL }}
+          slack-text: |
+            :info: *Cherry pick <${{ env.CHERRY_PICK_PR_URL }}|created> (`trunk` → `${{ needs.prepare.outputs.next_branch }}`)* Please ensure <${{ env.CHERRY_PICK_PR_URL }}|this cherry pick> is merged before releasing. Source PR: _<${{ env.SOURCE_PR_URL }}|#${{ env.SOURCE_PR_NUMBER }} - ${{ env.SOURCE_PR_TITLE }}>_.
+
+  # Handle failed cherry-pick to milestoned branch
+  handle-failures-milestoned:
+    needs: [prepare, cherry-pick-milestoned]
+    runs-on: ubuntu-latest
+    if: ${{ always() && needs.cherry-pick-milestoned.result == 'failure' }}
+    steps:
+      - name: Handle failure for milestoned branch
+        if: ${{ needs.cherry-pick-milestoned.outputs.status == 'failed' }}
+        run: echo "Processing failure for ${{ needs.prepare.outputs.milestoned_branch }}"
+
+      # Label original PR with failure tag
+      - name: Add "cherry pick failed" label
+        uses: actions/github-script@v7
+        with:
+          script: |
+            github.rest.issues.addLabels({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: parseInt('${{ needs.prepare.outputs.pr_number }}'),
+              labels: ['cherry pick failed']
+            })
+
+      # Comment on original PR about failure
+      - name: Comment on original PR
+        uses: actions/github-script@v7
+        env:
+          ERROR_MESSAGE: ${{ needs.cherry-pick-milestoned.outputs.error_message || 'Unknown error' }}
+          TARGET_BRANCH: ${{ needs.prepare.outputs.milestoned_branch }}
+          GITHUB_RUN_ID: ${{ github.run_id }}
+        with:
+          script: |
+            const pr = (await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: parseInt('${{ needs.prepare.outputs.pr_number }}')
+            })).data;
```

Comment:
Should we handle this request in a `try/catch` as we do in the `handle-successes-milestoned`?